apps/src/code-studio/activityUtils.js

```diff
@@ -2,34 +2,53 @@
  * A set of utility functions made for dealing with activities easier.
  */
 
+import { makeEnum } from '@cdo/apps/utils';
+
 /**
  * See ActivityConstants.
  */
 const MINIMUM_PASS_RESULT = 20;
 const MINIMUM_OPTIMAL_RESULT = 30;
 export const SUBMITTED_RESULT = 1000;
+export const LOCKED_RESULT = 1001;
```

Comment:
The ordering here has some implications: a higher value takes precedence over a lower value, and values below 1000 should't be cached by the client.  Is this the correct ordering?  We should also leave room for new values if you think there could ever be a status with higher precedence than `submitted` but lower precedence than `locked`.

It's an unfortunate system, but these values are written to the `Activity` and `UserLevel` table which we can't feasibly migrate.